Lombiq.Tests.UI.Shortcuts/ShortcutsFeatureIds.cs

```diff
@@ -7,9 +7,9 @@ public static class ShortcutsFeatureIds
     public const string Area = "Lombiq.Tests.UI.Shortcuts";
 
     public const string Default = Area;
-    public const string FeatureToggleTestBench = Default + ".FeatureToggleTestBench";
-    public const string MediaCachePurge = Default + ".MediaCachePurge";
-    public const string ShiftTime = Default + ".ShiftTime";
-    public const string Swagger = Default + ".Swagger";
-    public const string Workflows = Default + ".Workflows";
+    public const string FeatureToggleTestBench = $"{Default}.{nameof(FeatureToggleTestBench)}";
+    public const string MediaCachePurge = $"{Default}.{nameof(MediaCachePurge)}";
+    public const string ShiftTime = $"{Default}.{nameof(ShiftTime)}";
```

Comment:
I left the feature name "Shift Time". I think a verb is appropriate for features that do an action instead of add things, but chime in if you disagree.

Reply:
It'll be OK.